packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -0,0 +1,211 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'bottom_tab_bar.dart';
+
+/// Implements a tabbed iOS application's root layout and behavior structure.
+///
+/// The scaffold lays out the tab bar at the bottom and the content between or
+/// behind the tab bar.
+///
+/// A [tabBar] and a [tabBuilder] are required. The [CupertinoTabScaffold]
+/// will automatically listen to the provided [CupertinoTabBar]'s tap callbacks
+/// to change the active tab.
+///
+/// Tabs' contents are built with the provided [tabBuilder] at the active
+/// tab index. [tabBuilder] must be able to build the same number of
+/// pages as the [tabBar.items.length]. Inactive tabs will be moved [Offstage]
+/// and its animations disabled.
+///
+/// Use [CupertinoTab] as the content of each tab to support tabs with parallel
+/// navigation state and history.
+///
+/// ## Sample code
+///
+/// A sample code implementing a typical iOS information architecture with tabs.
+///
+/// ```dart
+/// new CupertinoTabScaffold(
+///   tabBar: new CupertinoTabBar(
+///     items: ...
```

Comment:
code in "Sample code" sections must be valid

Reply:
Ahhh that's what the analyzer was complaining about. I couldn't figure it out. Thanks